NebulaPatcher/Patches/Dynamic/EnemyUnitComponent_Patch.cs

```diff
@@ -0,0 +1,400 @@
+﻿#region
+
+using System;
+using HarmonyLib;
+using NebulaWorld;
+using UnityEngine;
+
+#pragma warning disable IDE1006
+#pragma warning disable IDE0007
+#pragma warning disable IDE0018
+
+#endregion
+
+namespace NebulaPatcher.Patches.Dynamic;
+
+[HarmonyPatch(typeof(EnemyUnitComponent))]
+internal class EnemyUnitComponent_Patch
+{
+    [HarmonyPostfix]
+    [HarmonyPatch(nameof(EnemyUnitComponent.GetTargetPosition_Ground))]
+    public static void GetTargetPosition_Ground_Postfix(int idType, ref Vector3 target, ref bool __result)
+    {
+        if (!Multiplayer.IsActive)
+        {
+            return;
+        }
```

Comment:
Minor nitpick, could be simplified to:
```suggestion
        if (!Multiplayer.IsActive) return;
```